v3/src/execution/index.ts

```diff
@@ -2,10 +2,164 @@ export {
   processGraphQLRequest,
   GraphQLRequestContext,
   GraphQLRequestPipelineConfig,
-} from "apollo-server-core/dist/requestPipeline";
-export { GraphQLRequest } from 'apollo-server-types';
+} from 'apollo-server-core/dist/requestPipeline';
+import { GraphQLRequest, VariableValues } from 'apollo-server-types';
+import {
+  DocumentNode,
+  parse,
+  GraphQLError,
+  GraphQLSchema,
+  validate,
+  execute,
+  ExecutionResult,
+  separateOperations,
+} from 'graphql';
+export { GraphQLRequest };
 
 // TODO(AS3) I'm not sure if this is execution.  Perhaps, a top-level export.
 export { GraphQLSchemaModule } from 'apollo-graphql';
 
 export { Context, ContextFunction } from 'apollo-server-core';
+
+/** Options for {@link processGraphQLRequest} */
+interface ProcessRequestInput {
+  request: GraphQLRequest;
+  schema: GraphQLSchema;
+}
+
+/**
+ * Process a GraphQLRequest. This includes parsing, validation,
+ * and execution of a query against a provided schema.
+ *
+ * @param args
+ * @param args.request - A GraphQLRequest object consisting of a query string and, optionally, an operationName, and variables
+ * @param args.schema - A GraphQLSchema to validate and execute the request against
```

Comment:
Note to self: this fn is missing the ability to pass in `context`, though it's supported in `executeGraphqlRequest()`. It should be passed through.

Reply:
Resolved in 7079636